test/test_experiment_parameters.py

```diff
@@ -21,6 +21,21 @@ def test_describe(self):
                 "type": "float"
             })
 
+    def test_evaluate_default(self):
+        def placeholder_get_default(key, default):
+            return key
+
+        param_float_default = IntParam("foo", "bar", 0., min=-1, max=1, unit="baz", scale=1)
```

Comment:
This should be `FloatParam`, presumably. I'd also remove all the `min`/`max`/`unit`/`scale` to put the focus on what is acutally being tested here (the default value).